rllib/policy/eager_tf_policy.py

```diff
@@ -697,8 +699,25 @@ def get_initial_state(self):
             return []
 
         @override(Policy)
-        def get_state(self):
+        def get_state(self) -> PolicyState:
+            # Legacy Policy state (w/o keras model and w/o PolicySpec).
             state = super().get_state()
+
+            # Add this Policy's spec so it can be retreived w/o access to the original
```

Comment:
Add PolicySpec AND - if possible - native keras/torch saved model files to returned state.

Reply:
PolicySpec is already part of the dict that is constructed by (newly renamed) RolloutWorker.get_state()
So I don't think we need to save these again? which will probably allow you to avoid a lot of the changes here.
btw, just a related note, we should try to use PolicySpec.serialize() whenever possible.

Reply:
done